### DISCUSSION TITLE
Possibility of multiple repilcals on single node (multi disk cluster)

### DISCUSSION THREAD

**Discussion by usma0118** [longhorn/longhorn#4461] [Q&A]
Just a question, is it possible to have multiple replicals on single node with multiple disks? 

my home server is single node but contains multiple disks, trying to see if i can have multiple replicas. for now i rely on daily snapshot and weekly backups for recovery.

**Comment by derekbit** [comment]
Yes, you can enable `Replica Node Level Soft Anti-Affinity` on `UI` > `Setting` > `General` page.
